parse_replays.py

```diff
@@ -91,6 +91,6 @@ def get_buttons(df):
 df = get_buttons(df)
 
 chdir(f'{args.output}/')
-df.to_csv(f'{args.name} + .csv')
+df.to_csv(f'{args.name}')
```

Comment:
Everything looks super good, tho I don't know much about web scraping at all. Definitely lots was done here. One thing that could help for future purposes is more documentation? Could be just a comment at the top of the main files explaining high-level what they do, or a short README

Reply:
^This